Composer/packages/server/src/models/asset/assetManager.ts

```diff
@@ -281,8 +282,11 @@ export class AssetManager {
     if (packageName) {
       return packageName
         .replace('generator-', '')
+        .replace('@microsoft/', '')
+        .replace('microsoft', '')
```

Comment:
this seems really brittle! we can make this more specific i think, at least like ^generator and ^@microsoft/.

Why the second removal of microsoft?